text/0732-cdk-cli-telemetry.md

```diff
@@ -222,6 +221,17 @@ It will help the team with:
 The second bullet point should improve the customer experience of reporting bugs that are
 affecting their work.
 
+While error messages, stack traces, and logs are essential to the debugging experience,
+we recognize that it can also contain sensitive information. Therefore we will NOT collect
+this data by default. We will still sanitize these values for customer content and
+personally-identifiable information as described below.
+
+You must opt-in for this data to be automatically sent to the telemetry service.
+Otherwise you will be given a choice to send this "error report" to the telemetry service when an  error occurs in the CLI.
+
+To opt-in, set `cli-telemetry-with-errors: true` in the context of your `cdk.json` or run
```

Comment:
Meh. Can we say that:

```json
{ 
  "cli-telemetry": "true"
}
```

Is the same as:

```json
{ 
  "cli-telemetry": {
    "usage": true,
    "errors": false,
  }
}
```

And then have `cdk cli-telemetry --enable-errors` set the nested key?